### PR TITLE
Update New-ScheduledTaskPrincipal.md LogonType info

### PR DESCRIPTION
**Pull request by PiJoCoder**
Submitted via Learn Editor.

### CODE REVIEW

**Comment by PiJoCoder** [comment]
#sign-off

File: docset/winserver2022-ps/scheduledtasks/New-ScheduledTaskPrincipal.md [hunk 1]
```diff
@@ -140,10 +140,12 @@ The acceptable values for this parameter are:
 - None
 - Password
 - S4U
-- Interactive
+- - Interactive
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/scheduledtasks/New-ScheduledTaskPrincipal.md:143]:
```suggestion
- Interactive
```

File: docset/winserver2022-ps/scheduledtasks/New-ScheduledTaskPrincipal.md [hunk 2]
```diff
@@ -140,10 +140,12 @@ The acceptable values for this parameter are:
...
 - Group
 - ServiceAccount
 - InteractiveOrPassword
+   
+For more information on LogonType values, see [Principal.LogonType](/windows/win32/taskschd/principal-logontype#property-value)
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/scheduledtasks/New-ScheduledTaskPrincipal.md:148]:
```suggestion
For more information about LogonType values, see [Principal.LogonType](/windows/win32/taskschd/principal-logontype#property-value)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PiJoCoder, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by PiJoCoder** [comment]
#sign-off

**Comment by PiJoCoder** [comment]
#sing-off

**Comment by PiJoCoder** [comment]
@JohanFreelancer9 do you know how we move this PR forward?

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 do you know how we move this PR forward?

@scanum, please assist.

**Comment by scanum** [comment]
@sdwheeler 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by sdwheeler — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd Hope all is well. Could you please help us merge? Thanks!

**Pull request merged**